head_pose_from_webcam.py

```diff
@@ -6,13 +6,19 @@
 import argparse
 import numpy as np
 
-#import Face Recognition
-import face_recognition
+#import Face Recognition libraries
+import mediapipe as mp
```

Comment:
if we use mediapipe, there's no need to use dlib for shape predicting. We can use mediapipe's face mesh for landmarks. Let's merge it when you get rid of Dlib.
Thanks for your contribution. Please let me know if you need more information. Great job by the way.